README.md

```diff
@@ -1,35 +1,35 @@
-Ghub.el — Minuscule client library for the Github API
+Ghub.el — Miniscule client library for the GitHub API
 =====================================================
 
-Ghub is a library that provides basic support for using the Github
+Ghub is a library that provides basic support for using the GitHub
```

Comment:
I know that the company spells its name that way, but I prefer not to have any capital letters appear in the middle of words.